packages/@expo/metro-config/src/transform-plugins/import-export-plugin.ts

```diff
@@ -228,8 +228,8 @@ export function importExportPlugin({ types: t }: { types: Types }): PluginObj<St
                   {
                     const properties = d.id.properties;
                     properties.forEach((p) => {
-                      // @ts-expect-error Property 'value' does not exist on type 'RestElement'
-                      const name = p.value.name;
+                      // @ts-expect-error Property 'name' does not exist on type 'ArrayExpression'.
```

Comment:
Do we have a nicer way to bail here? Might be worth to leave TODO comments here if the stack will leave behind `@ts-expect-error`s. Alternatively, if it's hard to get full coverage of all possibilities, we could:
- decide if we want to bail gracefully somehow (if possible)
- add a `debug` log here, so this is spottable in debug logs, if requested from users